### DISCUSSION TITLE
STIGMANAGER Environment Variables

### DISCUSSION THREAD

**Discussion by lp177344** [NUWCDIVNPT/stig-manager#510] [Feedback Requested]
Hello, 

I have opted to install STIGMANAGER from source code on to Windows Server 2019. 

I continue to run into an error when running the command "node index.js"

"[DB] Access denied for user 'stigman'@'localhost' (using password: NO)"

I am rechecking my steps, but can someone assist me in locating the environment variable config file location? I assume it is here: ..\Desktop\stig-manager-main\docs\installation-and-setup\environment-variables.rst

thank you!

**Comment by cd-rite** [comment]
Hi @lp177344
STIGMan does not use a config file. You must create or set the environment variables on your host system (you could create your own file/script that sets them for you, or for use with Docker, if you go that route, but that  will be specific to your situation). The file you are referencing is just our documentation describing the different environment variables you can use to affect application behavior. 

Here's a quick document from Oracle about how to do this in windows. The actual environment variables you create would be different, they would be from the list you mention above: https://docs.oracle.com/en/database/oracle/machine-learning/oml4r/1.5.1/oread/creating-and-modifying-environment-variables-on-windows.html

Hope that helps!

**Reply by lp177344** [reply]
that worked like a charm! thank you for the quick reply!
